src/libs/actions/IOU.js

```diff
@@ -736,6 +746,10 @@ function getMoneyRequestInformation(
           }
         : undefined;
 
+    // The policy expense chat should have the GBR only when its a paid policy and the scheduled submit is turned off
+    // so the employee has to submit to tits manager manually.
+    const hasOutstandingChildRequest = isPolicyExpenseChat && isFromPaidPolicy && needsToBeManuallySubmitted;
```

Comment:
NAB: based on my comment above
```suggestion
    const hasOutstandingChildRequest = isPolicyExpenseChat && needsToBeManuallySubmitted;
```